src/vm/arm64/stubs.cpp

```diff
@@ -948,8 +929,8 @@ void JIT_TailCall()
 
 extern "C" void * ClrFlsGetBlock()
 {
-    _ASSERTE(!"ARM64:NYI");
-    return NULL;
+    // not required for coreclr
+    throw "ClrFlsGetBlock not yet implemented";
```

Comment:
yes will try...I think it was cascading to a lot of ifdef...not sure if it was this one.